packages/html/src/switch/switch.spec.tsx

```diff
@@ -11,6 +11,7 @@ const states = [
 
 const options = {
     size: [ Size.small, Size.medium, Size.large ],
+    rounded: [ null, Roundness.small, Roundness.medium, Roundness.large, Roundness.full ]
```

Comment:
do not use null here, but rather in the templates same as the button. 
Btw, the label for the same option should be 'none', as that is what the component API name is.